### PR TITLE
sWaves add fee/revenue

### PR DESCRIPTION
**Pull request by stupid-boar**
- [sWaves official site](https://swaves.pepe.team)
- [sWaves defillama](https://defillama.com/protocol/pepeteam-swaves)
- [sWaves token info](https://bridge.pepe.team/tokens/WAVES-Waves-PPT)

### CODE REVIEW

**Comment by llamatester** [comment]
The pepe-swaves adapter exports: 
        
 
 _______________________________________
    Fees for 1/1/2024
    _______________________________________
    
    WAVES 👇
    Backfill start time: 31/10/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1704153598
    Daily fees: 368.47529999999995
    Daily revenue: 9.2118825
    Daily holders revenue: 0
    
    
    
    
    Done in 13.29s.

**Comment by dtmkeng** [comment]
it fees all for chain waves or just sWaves ?

**Comment by dtmkeng** [comment]
adapter name `pepe-swaves` is just protocol not chain

File: fees/pepe-swaves/index.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+};
+
+const adapter: Adapter = {
+    adapter: {
+        [CHAIN.WAVES]: {
+            fetch,
+            start: async () => 1667250000 // Mon Oct 31 2022 21:00:00 GMT+0000
+        },
+    },
+    protocolType: ProtocolType.CHAIN
```
Comment (dtmkeng) [inline fees/pepe-swaves/index.ts]: could you remove this if you need list `pepe-swaves` not fees all of waves chain

**Comment by llamatester** [comment]
The pepe-swaves adapter exports: 
        
 
 _______________________________________
    Fees for 1/1/2024
    _______________________________________
    
    WAVES 👇
    Backfill start time: 31/10/2022
    NO METHODOLOGY SPECIFIED
    Timestamp: 1704153598
    Daily fees: 368.47529999999995
    Daily revenue: 9.2118825
    Daily holders revenue: 0
    
    
    
    
    Done in 17.71s.

File: fees/pepe-swaves/index.ts (hunk 1 above)
Comment (stupid-boar) [reply fees/pepe-swaves/index.ts]: @dtmkeng sorry, you're right. My mistake, fixed

**Pull request merged**